frontend/src/views/CompetingSpecies/CompetingSpecies.vue

```diff
@@ -357,11 +371,11 @@ export default {
       try {
         const path = "http://localhost:5000/CompetingSpecies/AllPresets";
         const payload = {
-          userEmail: this.$store.state.activeUser.email
+          userEmail: this.activeUser.email
```

Comment:
```suggestion
          userEmail: this.activeUser.email,
```

consistency with commas at EOL object entries (formatter will sort)